src/framework/uicomponents/qml/MuseScore/UiComponents/SearchField.qml

```diff
@@ -38,5 +38,6 @@ TextInputField {
     hint: qsTrc("uicomponents", "Search")
     hintIcon: IconCode.SEARCH
 
-    clearTextButtonVisible: Boolean(searchText)
+    clearTextButtonVisible: searchText ?? null
+    clearTextButton.toolTipTitle : qsTrc("uicomponents", "Clear")
```

Comment:
please, add toolTipTitle directly in TextInputField.qml for clearTextButton instead of setting 
`accessible.name: qsTrc("global", "Clear")`

Reply:
Yeah done